src/stactools/global_mangrove_watch/stac.py

```diff
@@ -299,6 +299,17 @@ def create_item(
         transform=rasterio.transform.from_bounds(*item_attributes["bbox"], *ITEM_SHAPE),
     )
 
+    # ensure proj:epsg gets set (pystac bug)
+    item.properties["proj:epsg"] = EPSG
+
+    # we need to stick with projection extension <v2.0 in order to maintain
+    # compatibility with the STACIT driver in older GDAL versions
+    for i, extension in enumerate(item.stac_extensions):
+        if "projection/v2.0" in extension:
+            _extension = item.stac_extensions.pop(i)
+            item.stac_extensions.append(_extension.replace("v2.0.0", "v1.2.0"))
+            _ = item.properties.pop("proj:code")
```

Comment:
This future-proofs if proj comes out with a v2.X.Y by just forcing a single `projection` with a known working version:

```suggestion
    extensions = ["https://stac-extensions.github.io/projection/v1.2.0/schema.json"]
    extensions.extend((e for e in item.stac_extensions if not e.startswith("https://stac-extensions.github.io/projection"))
    item.stac_extensions = extensions
    _ = item.properties.pop("proj:code", None)
```

Reply:
oh, great catch - thanks!